charts/prometheus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: prometheus
-appVersion: 2.39.1
+appVersion: 2.39.2
```

Comment:
Hi @zanhsieh, I think you accidentally bumped `appVersion`. Latest prometheus is 2.39.1 and 2.39.2 doesn't exist.

Reply:
```suggestion
appVersion: 2.39.1
```